resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DeletedRuntimeChecker.scala

```diff
@@ -19,38 +18,38 @@ object DeletedRuntimeChecker {
     new CheckRunner[F, Runtime] {
       override def appName: String = resourceValidator.appName
 
-      override def configs = CheckRunnerConfigs(s"deleted-runtime", true)
+      override def configs = CheckRunnerConfigs(s"deleted-runtime", shouldAlert = true)
```

Comment:
is it just some renaming and function moved around in this file?

Reply:
Yes, no functional change.